robottelo.properties.sample

```diff
@@ -106,7 +106,9 @@ ssh_key=
 # be pulled from there instead of using the CDN channel. These information is
 # more suited to be used for downstream, downstream-iso and zstream builds.
 # capsule_repo=http://capsule/repo
-# sattools_repo=http://sattools/repo
+# sattools_repo=
```

Comment:
I was just reading the comment you said about using dicts and was writing never mind here.